Code/Engine/Foundation/Memory/Policies/AllocPolicyStack.h

```diff
@@ -3,11 +3,12 @@
 #include <Foundation/Containers/HybridArray.h>
 
 /// \brief This allocation policy works like a stack. You can only "push" and "pop" allocations
```

Comment:
Btw this is not a stack allocation policy but a linear allocation policy. The name is misleading and the description is just wrong.

Reply:
Renamed it and fixed the description.